pages/resources/testnet/upgrades/v27.md

```diff
@@ -0,0 +1,47 @@
+# Testnet-1 upgrade: v0.27
+
+import Callout from 'nextra-theme-docs/callout'
+
+Instructions for `2022-Nov-16` testnet upgrade to axelar-core `v0.27.0`, vald `v0.27.0`, and tofnd `v0.10.1`.
+Release can be found [here](https://github.com/axelarnetwork/axelar-core/releases/tag/v0.27.0).
+
+Upgrade height `3842800` [countdown](https://testnet.mintscan.io/axelar-testnet/blocks/3842800)
```

Comment:
TODO
Need to update this block height once upgrade proposal is passed on testnet